apps/nextjs/src/components/AppComponents/AdditionalMaterials/StepLayouts/StepFour.tsx

```diff
@@ -177,6 +179,11 @@ const StepFour = ({ handleRefineMaterial }: StepFourProps) => {
     );
   }
 
+  if (isDownloading && !userHasSeenSurvey) {
+    setUserHasSeenSurvey(true);
+    setDialogWindow("additional-materials-user-feedback");
+  }
```

Comment:
Should this be in an effect? You could run the risk of over-calling it, or getting into a render loop